src/wp-includes/blocks.php

```diff
@@ -219,7 +219,11 @@ function register_block_style_handle( $metadata, $field_name, $index = 0 ) {
 	// Check whether styles should have a ".min" suffix or not.
 	$suffix = SCRIPT_DEBUG ? '' : '.min';
 	if ( $is_core_block ) {
-		$style_path = "style$suffix.css";
+		if ( 'editorStyle' === $field_name ) {
+			$style_path = "editor$suffix.css";
+		} else {
+			$style_path = "style$suffix.css";
+		}
```

Comment:
Consider a ternary here:
```suggestion
		$style_path = ( 'editorStyle' === $field_name ) ? "editor$suffix.css" : "style$suffix.css";
```